src/pages/Withdraw.tsx

```diff
@@ -48,56 +67,32 @@ function Withdraw({ poolName }: Props): ReactElement {
       ) {
         return
       }
-      const tokenInputSum = parseUnits(
-        POOL.poolTokens
-          .reduce(
-            (sum, { symbol }) =>
-              sum + (+withdrawFormState.tokenInputs[symbol].valueRaw || 0),
-            0,
-          )
-          .toString(),
-        18,
-      )
-      let withdrawLPTokenAmount
       if (poolData.totalLocked.gt(0) && tokenInputSum.gt(0)) {
+        const withdrawTokenAmounts = POOL.poolTokens.map(
+          (token) => withdrawFormState.tokenInputs[token.symbol].valueSafe,
+        )
         if (isLegacySwapABIPool(poolData.name)) {
-          withdrawLPTokenAmount = await (
+          const tokenAmount = await (
```

Comment:
Maybe rename to "calculatedTokenAmount" to add a little more specificity

Reply:
Updated!